### ISSUE TITLE
[Sourcing][Accessibility]  select box(and other components) focus issue after selecting element

### ISSUE THREAD

**Issue by rayansailani** [SAP/fundamental-ngx#9667]
#### Is this a bug, enhancement, or feature request?

Bug

#### Briefly describe your proposal.
In the fundamental suite of components fd-select, fdp-select and all the other components mentioned below have an issue with the focus. When the user makes a selection in the select box either using the keyboard or the mouse after the selection is made the focus should go back to the select component. 

Current behaviour: The focus is lost and is not seen on any visible component. 

Expected behaviour: The focus should be moved back to the selection component. This expected behaviour is currently displayed in SAP UI5 component library. 

**affected components: 
fd-select
fd-combobox
fd-multicombobox
fd-multinput
fdp-select
fdp-combobox
fdp-multicombobox
fdp-multinput**

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

0.40.1-rc.3 and previous versions as well. 

#### If this is a bug, please provide steps for reproducing it.

In a select component like fdp-select or fd-select (let's say https://fundamental-ngx.netlify.app/#/core/select) use keyboard or mouse to select an element. Once done, you would expect that the focus is back on the parent component but it's currently lost on an invisible component.

**Comment by I543348** [comment]
@rayansailani  pls check and add your inputs.
Thanks.

**Comment by rayansailani** [comment]
Evaluated the fixes on the following component and they seem to be working fine. 
fd-select
fd-combobox
fd-multicombobox
fd-multinput
fdp-select
fdp-combobox
fdp-multicombobox
fdp-multinput

Thanks for the fix.
